managedtenants/core/addons_loader/sss.py

```diff
@@ -182,5 +185,17 @@ def _walk_sss(data):
             sss["spec"]["resources"][kind].append((name, resource))
         return sss
 
+    @staticmethod
+    def _walk_template(data):
+        template = deepcopy(data)
+        old_resources = deepcopy(data["objects"][0]["spec"]["resources"])
+        template["objects"][0]["spec"]["resources"] = defaultdict(list)
+        for resource in old_resources:
+            kind = resource["kind"]
+            name = resource["metadata"]["name"]
+            template["objects"][0]["spec"]["resources"][
+                kind].append((name, resource))
+        return template
+
```

Comment:
```suggestion
```
In favor of https://github.com/mt-sre/managed-tenants-cli/pull/136/files#r847972475